apps/example-nodejs-jwt/app.js

```diff
@@ -75,12 +77,13 @@ app.get("/", function (req, res) {
        <li><%= role %></li>
      <% }); %>
    </ul>
+     <p> Your JSON Web Token (JWT) was verified against <a href="<%= proxyAddr %><%= jwtResource %>"><%= proxyAddr %><%= jwtResource %></a> </p>
```

Comment:
I think it's better to provide a link back to root cluster, but I've kept in the jwks url but removed the hyperlink for it. 
```suggestion
     <p> Your JSON Web Token (JWT) was sent via <a href="<%= proxyAddr %>"><%= proxyAddr %></a> and verified against <%= proxyAddr %><%= jwtResource %></p>
```